src/templates/ja/2020/index.html

```diff
@@ -8,16 +8,17 @@
 {% block date_published %}2019-11-04T12:00:00.000Z{% endblock %}
 {% block date_modified %}2020-07-06T00:00:00.000Z{% endblock %}
 
-{% block intro_title %}Coming soon!{% endblock %}
-{% block intro_sub_title %}The 2020 <b>state of the web</b> report{% endblock %}
+{% block intro_title %}もうすぐです！{% endblock %}
+{% block intro_sub_title %}2020の<b>web状況</b>報告{% endblock %}
 
 {% block mission %}
 <p>
-  Experts from around the web community are currently hard at work planning, analyzing, and writing content for the 2020 edition of the Web Almanac, which is on track to be published at the end of the year.
+  現在、ウェブ・コミュニティの専門家たちが、年末に発行を予定しているWeb Almanac 2020年版に向けて、企画、分析、コンテンツの執筆に奔走しています。
 </p>
 
 <p>
   There are many opportunities available for new contributors to join the project, including: authors, reviewers, analysts, editors, translators, and developers. If you're interested in contributing to the 2020 edition, we'd love to have you!
```

Comment:
```suggestion
  There are many opportunities available for new contributors to join the project, including: authors, reviewers, analysts, editors, translators, and developers. If you're interested in contributing to the 2020 edition, we'd love to have you!
```
You've left the English behind. Can you delete this?

Reply:
As this is the only issue, will go ahead and merge this and fix myself.